packages/server/src/auth/dto/auth.dto.ts

```diff
@@ -53,9 +54,19 @@ export class EmailLoginDto {
 }
 
 export class SignupTransformPipe implements PipeTransform {
-  transform(body: any): UserSignupDto {
+  transform(body: UserSignup): UserSignupDto {
     const user = new UserSignupDto();
-    user.projectId = body.project_id.toString();
+    const properties = ['projectId', 'username', 'email', 'password'];
```

Comment:
Pull main and merge into this branch. I replaced these checks with `ValidationPipe`

Reply:
merged